tests/integration/apigateway/test_apigateway_dynamodb.py

```diff
@@ -45,10 +44,9 @@ def test_rest_api_to_dynamodb_integration(
     table_name = table["TableName"]
 
     # insert items
-    dynamodb_table = dynamodb_resource.Table(table_name)
     item_ids = ("test", "test2", "test 3")
     for item_id in item_ids:
-        dynamodb_table.put_item(Item={"id": item_id})
+        aws_client.dynamodb.put_item(TableName=table_name, Item={"id": {"S": item_id}})
```

Comment:
The client needs the items to be typed. The Boto resource had an intrinsic [type conversion](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/customizations/dynamodb.html#valid-dynamodb-types) so it wasn't necessary earlier